packages/flutter_test/lib/src/binding.dart

```diff
@@ -494,9 +494,25 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   ///
   /// When [handlePointerEvent] is called directly, [pointerEventSource]
   /// is [TestBindingEventSource.device].
+  ///
+  /// This default means that pointer events triggered by the [WidgetController]
```

Comment:
nit: suggestion for clearity:

```suggestion
  /// This means that pointer events triggered by the [WidgetController] (e.g. via [WidgetController.tap])
```